tests/functional/RemoteWebDriverCreateTest.php

```diff
@@ -40,7 +42,12 @@ public function testShouldStartBrowserAndCreateInstanceOfRemoteWebDriver()
 
         // MicrosoftEdge on Sauce Labs started to identify itself back as "msedge"
         if ($this->desiredCapabilities->getBrowserName() !== WebDriverBrowserType::MICROSOFT_EDGE) {
-            $this->assertSame($this->desiredCapabilities->getBrowserName(), $returnedCapabilities->getBrowserName());
+            if ($this->desiredCapabilities->getBrowserName() === WebDriverBrowserType::SAFARI) {
+                // TODO: ???
```

Comment:
safaridriver is returning `Safari` (Capital) vs `safari` expected

@OndraM what's the purpose of this assert?

Reply:
used `assertEqualsIgnoringCase`